app/controllers/users_controller.rb

```diff
@@ -8,6 +8,15 @@ class UsersController < ActionController::Base
         :email_exists => "email already exists"
     }
 
+    def matches
+        user = User.find_by_id(params[:id])
+        if user.present?
+            render :json => user.matches
+        else
+            render :json => []
```

Comment:
Returning an empty list is probably fine, but just as an aside, the convention we followed for the other API calls is `{ :error => "Some error message" }` if something went wrong. Maybe we could refactor this to return a hash containing "error" if there's an error, else `{ "matches": [ ... ]  }`